src/plotViewer/index.ts

```diff
@@ -515,9 +522,13 @@ export class HttpgdViewer implements IHttpgdViewer {
         this.state = await this.api.getState();
         if(this.state.upid !== oldUpid){
             await this.refreshPlots();
+            if (isHost()) {
+                void shareBrowser(shareUrl, '[VSC-R] R Plot', true);
+                void rHostService.notifyGuestPlotManager(shareUrl);
```

Comment:
Instead of `shareUrl` can we use `this.host`? This would be safer, since there can be multiple Httpgd viewers at the same time and we'd avoid an (unnecessary?) global variable.

Reply:
It seems that `this.host` isn't a valid url to share, but `http://${this.host}/live?token=${this.token}` works